templates/course/course.html

```diff
@@ -1,26 +1,26 @@
 <!-- course.html -->
 
 
-<form method="GET">
+<form  method="GET">
 　要查詢的關鍵字: <input type="text" name="keyword">
 　<input type="submit" value="搜尋">
 </form>
 
 <table border="1" height="35">
 		<tr>
-			<td width="100"><font color="darkpurple"><b>課程名稱</b></font></td>
-			<td width="20"><font color="darkpurple"><b>學分</b></font></td>
-			<td width="50"><font color="darkpurple"><b>全年/半年</b></font></td>
-			<td width="50"><font color="darkpurple"><b>必修/選修</b></font></td>
-			<td width="100"><font color="darkpurple"><b>授課教師</b></font></td>
-			<td width="20"><font color="darkpurple"><b>加選方式</b></font></td>
-			<td width="100"><font color="darkpurple"><b>上課時間</b></font></td>
-			<td width="100"><font color="darkpurple"><b>上課地點</b></font></td>
-			<td width="30"><font color="darkpurple"><b>總人數</b></font></td>
-			<td width="20"><font color="darkpurple"><b>點擊次數</b></font></td>
-			<td width="100"><font color="darkpurple"><b>選課限制條件</b></font></td>
-			<td width="100"><font color="darkpurple"><b>備註</b></font></td>
-			<td width="100"><font color="darkpurple"><b>課程網址</b></font></td>
+			<td width="100"><b class="title">課程名稱</b></font></td>
+			<td width="20"><b class="title">學分</b></font></td>
+			<td width="50"><b class="title">全年/半年</b></font></td>
+			<td width="50"><b class="title">必修/選修</b></font></td>
+			<td width="100"><b class="title">授課教師</b></font></td>
+			<td width="20"><b class="title">加選方式</b></font></td>
+			<td width="100"><b class="title">上課時間</b></font></td>
+			<td width="100"><b class="title">上課地點</b></font></td>
+			<td width="30"><b class="title">總人數</b></font></td>
+			<td width="20"><b class="title">點擊次數</b></font></td>
+			<td width="100"><b class="title">選課限制條件</b></font></td>
+			<td width="100"><b class="title">備註</b></font></td>
+			<td width="100"><b class="title">課程網址</b></font></td>

```

Comment:
```suggestion
			<td width="100"><strong class="title">課程網址</strong></td>
```
Do not use `<b>` tags. Use `<strong>` or `<em>` tags instead. Also don't forget to delete closing `</font>` tags.